plugins/inventory/kubevirt.py

```diff
@@ -317,6 +320,28 @@ def get_port_from_service(service: Dict) -> Optional[str]:
 
         return None
 
+    @staticmethod
+    def is_windows(guest_os_info: Dict, annotations: Dict) -> bool:
+        if (id := guest_os_info.get("id")) is not None:
+            return id == "mswindows"
```

Comment:
wdyt about making the id `mswindows` a const?

Reply:
Created a constant for it.